gradle.properties

```diff
@@ -2,3 +2,6 @@ group = com.enonic.app
 projectName = gitpull
 xpVersion=7.0.0
 version=3.0.0
+
+# Version of dependencies
+jgitVersion=5.9.0.202009080501-r
```

Comment:
Don't use properties for that. Define directly in gradle file

- We use properties file in release script
- Dependabot gets confused